src/sprout/Helpers/Fb.php

```diff
@@ -598,8 +599,25 @@ public static function chunkedUpload($name, array $attrs = [], array $params = [
 
         if (is_string($friendly_vals)) {
```

Comment:
So we've got this handling single uploads, is that right?

Can we support multiple files with `if (is_array($friendly_vals))` in the same way? 

I suspect we'll have to filter out fields that already exist in the files array from the array zip above.